### DISCUSSION TITLE
Upscale issue. Files are HUGE

### DISCUSSION THREAD

**Discussion by AFKevin89** [upscayl/upscayl#516] [Q&A]
Hey there..

im tryinge to upscale some midjourney pictures (1024x1024). After theyre upscaled the are 4096x4096 and they are ALL exactly 65.649kb. Everyone got the same size. Its weird.. Since the update i guess

Any one know a sollution?

thanks from germany :)

**Comment by begin-theadventure** [comment]
Have you tried `Settings`->`IMAGE COMPRESSION`?

**Reply by AFKevin89** [reply]
But i didnt need to do that before the update.. and i dont want it to be compressed?!

**Reply by aaronliu0130** [reply]
Pretty sure sizes were  also this way before the update.

**Reply by AFKevin89** [reply]
tehy were not.. when i upscaled a 1MB file it typically goes up to like ~10MB. Not 65MB.
especially not all exactly 65.649kb on every file regardless of the original size

And it doesnt do it always.. i just made batch that was fine.

**Reply by AFKevin89** [reply]
![image](https://github.com/upscayl/upscayl/assets/147416268/203aa144-f195-409e-81ca-371696a77540)
 for example: these are upscaled from around 1mb original size.

But most of the time it wont do it properly and they and up all being 65mb

**Reply by aaronliu0130** [reply]
Ah, I thought your dot was a decimal. What do the exactly 65.649 KB files look like?

**Reply by AFKevin89** [reply]
they look totally normal. Just Huge in size. But still just 4096x4096

**Reply by aaronliu0130** [reply]
Have you tried upscayling some of your images that would result in 65 MB in an old version?

**Reply by AFKevin89** [reply]
I tried to find older Versions.. But i think im to dumb. 
I dont really know how to navigate Github :D

**Reply by aaronliu0130** [reply]
https://github.com/upscayl/upscayl/releases/tag/v2.5.5

**Reply by AFKevin89** [reply]
looks like an older version fixed the problem

**Reply by aaronliu0130** [reply]
That's very weird. Do you notice any weird things in the new version's logs?

**Reply by AFKevin89** [reply]
No i never even looked in the logs.. didnt even knew there are logs :D  
But i also switched from an Nvidia RTX 2060 to an AMD RX 6700 XT...
Maybe that was the cause?!
But older version is good to use for me...

**Reply by aaronliu0130** [reply]
Welp, we still want to figure it out for new versions though.

**Reply by NayamAmarshe** [reply]
You can disable post processing in new versions.
This issue is probably because of sharp.js compression being different than the regular one.

**Reply by AFKevin89** [reply]
Ill reinstall the newest in a few days and let you know what the logs say so you can work on it :)

Does anyone know if there is a free software like upsayl but for removing backgrounds ib bulk? That is the only thing i need to pay for at the moment and it would be awesome if there is a software for it.

**Reply by aaronliu0130** [reply]
I have no idea.

**Comment by AFKevin89** [comment]


**Reply by aaronliu0130** [reply]
Yes.
